crates/dojo-lang/src/plugin_test_data/system

```diff
@@ -454,15 +483,131 @@ mod MoveSystem {
+            }.get('Player', player_id.into(), 0_u8, 0_usize);
+            let __player_position_player = serde::Serde::<Player>::deserialize(
+                ref __player_position_player_raw
+            );
+            let player_position = (__player_position_position, __player_position_player);
+            if foo.len() > 0_u32 {
+                let positions_query = ArrayTrait::<usize>::new();
+                let __positions_query_position_ids = IWorldDispatcher {
+                    contract_address: world_address
+                }.entities('Position', 0);
+
+                let __positions_query_player_ids = IWorldDispatcher {
+                    contract_address: world_address
+                }.entities('Player', 0);
+            }
+        } else if bar == 321 {
+            let mut __player_position_position_raw = IWorldDispatcher {
+                contract_address: world_address
+            }.get('Position', player_id.into(), 0_u8, 0_usize);
+            let __player_position_position = serde::Serde::<Position>::deserialize(
+                ref __player_position_position_raw
+            );
+
+            let mut __player_position_player_raw = IWorldDispatcher {
+                contract_address: world_address
+            }.get('Player', player_id.into(), 0_u8, 0_usize);
+            let __player_position_player = serde::Serde::<Player>::deserialize(
+                ref __player_position_player_raw
+            );
+            let player_position = (__player_position_position, __player_position_player);
+        } else {
+            let mut __player_position_position_raw = IWorldDispatcher {
+                contract_address: world_address
+            }.get('Position', player_id.into(), 0_u8, 0_usize);
+            let __player_position_position = serde::Serde::<Position>::deserialize(
+                ref __player_position_position_raw
+            );
+
+            let mut __player_position_player_raw = IWorldDispatcher {
+                contract_address: world_address
+            }.get('Player', player_id.into(), 0_u8, 0_usize);
+            let __player_position_player = serde::Serde::<Player>::deserialize(
+                ref __player_position_player_raw
+            );
+            let player_position = (__player_position_position, __player_position_player);
+        }
+        let mut __player_position_position_raw = IWorldDispatcher {
+            contract_address: world_address
+        }.get('Position', player_id.into(), 0_u8, 0_usize);
+        let __player_position_position = serde::Serde::<Position>::deserialize(
+            ref __player_position_position_raw
+        );
+
+        let mut __player_position_player_raw = IWorldDispatcher {
+            contract_address: world_address
+        }.get('Player', player_id.into(), 0_u8, 0_usize);
+        let __player_position_player = serde::Serde::<Player>::deserialize(
+            ref __player_position_player_raw
+        );
```

Comment:
it looks like we lost the scoped block here

Reply:
Ah nice catch